bbb-common-web/src/main/java/org/bigbluebutton/presentation/UploadedPresentation.java

```diff
@@ -177,4 +179,27 @@ public boolean isUploadFailed() {
   public ArrayList<String> getUploadFailReason() {
     return uploadFailReason;
   }
+
+  public String getPresentationConvertedName() {
+    if (presentationConvertedName != null) {
+      return presentationConvertedName;
+    } else {
+      return "";
+    }
+  }
+
+  public void generatePresentationConvertedName() {
+    String nameWithoutExtension = FilenameUtils.removeExtension(name);
+    this.presentationConvertedName = nameWithoutExtension.concat(".pdf");;

```

Comment:
Consider receiving the extension as a parameter of this function.
Instead of ".pdf" fixed.